src/components/User/UserDetailComponent.jsx

```diff
@@ -22,65 +24,50 @@ const assetsAssigned = (allocatedAssets) => {
     <Table basic selectable>
       <Table.Header>
         <Table.Row>
-          <Table.HeaderCell>
-            Asset Type
-          </Table.HeaderCell>
-          <Table.HeaderCell>
-            Asset Code
-          </Table.HeaderCell>
-          <Table.HeaderCell>
-            Serial Number
-          </Table.HeaderCell>
+          <Table.HeaderCell>Asset Type</Table.HeaderCell>
+          <Table.HeaderCell>Asset Code</Table.HeaderCell>
+          <Table.HeaderCell>Serial Number</Table.HeaderCell>
         </Table.Row>
       </Table.Header>
       <Table.Body>
-        {
-          allocatedAssets.map(asset => (
-            <TableRowDetail
-              key={asset.id}
-              data={asset}
-              viewDetailsRoute={`/assets/${asset.uuid}/view`}
-              headings={[
-                'asset_type',
-                'asset_code',
-                'serial_number'
-              ]}
-            />
-          ))
-        }
+        {allocatedAssets.map(asset => (
+          <TableRowDetail
+            key={asset.id}
+            data={asset}
+            viewDetailsRoute={`/assets/${asset.uuid}/view`}
+            headings={['asset_type', 'asset_code', 'serial_number']}
+          />
+        ))}
       </Table.Body>
     </Table>
   );
 };
 
 const UserDetailComponent = (props) => {
-  if (props.isLoading) {
-    return (
-      <LoaderComponent />
-    );
-  }
-
   if (isEmpty(props.userDetail)) {
-    return (
-      <div>
-        User Not Found
-      </div>
-    );
+    return <div>User Not Found</div>;
   }
 
   return (
     <div>
       <Header as="h2" textAlign="left">
         {props.userDetail.full_name || 'Andelan'}
       </Header>
+      {
+        (props.successMessage !== '' || props.errorMessage !== '') && (
+          <StatusMessageComponent
```

Comment:
The success Message only displays ones and never displays if another action is dispatched. Please fix this.

Reply:
Fixed now 😁